pkgs/desktops/gnome/default.nix

```diff
@@ -259,6 +259,14 @@ lib.makeScope pkgs.newScope (self: with self; {
   gnome-autoar = callPackage ./misc/gnome-autoar { };
 
   gnome-packagekit = callPackage ./misc/gnome-packagekit { };
+
+  #### Wallpapers -- 3rd party wallpapers that specifically use Gnome's XML format.
+  
+  wallpaper-lakeside = callPackage ./artwork/lakeside { };
+
+  wallpaper-firewatch = callPackage ./artwork/firewatch { };
```

Comment:
Please do not add more new packages to `gnome` namespace, we are slowly dissolving it. Put them in `pkgs/top-level/all-packages.nix`.